gbpservice/nfp/bin/startup/redhat/nfp_proxy.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=One Convergence's NFP Proxy
+After=syslog.target network.target
+
+[Service]
+ExecStart=source /opt/stack/gbp/devstack/lib/nfp;namespace_delete /home/stack/devstack;namespace_create /home/stack/devstack 11.0.0.3
```

Comment:
Even in the multi-node installation, configurator image will be built and this IP will be replaced. We should keep some string in this place for Ravi to replace during installation script. Lets call this string "configurator-ip"